www/scripts/codecheckerviewer/CommentView.js

```diff
@@ -0,0 +1,272 @@
+    } else if (delta < day * 2) {
+      fuzzy = 'yesterday';
+    } else if (delta < week) {
+      fuzzy = Math.floor(delta / day) + ' days ago.';
+    } else if (delta < day * 8) {
+      fuzzy = '1 week ago.';
+    } else if (delta < month) {
+      fuzzy = Math.floor(delta / week) + ' weeks ago.';
+    } else {
+      fuzzy = 'on ' + locale.format(date, "yyyy-MM-dd HH:mm");
+    }
+
+    return fuzzy;
+  }
+
+  var Reply = declare(ContentPane, {
+    constructor : function () {
+      var that = this;
+
+      this._content  = new SimpleTextarea({
+        class       : 'reply-content',
+        placeholder : 'Leave a message...',
+        rows        : 4
+      });
+
+      this._commentBtn  = new Button({
+        class   : 'reply-btn',
+        label   : 'Comment',
+        onClick : function () {
+          var commentData = new CC_OBJECTS.CommentData({
+            author    : 'Anonymous', // TODO: this should be replaced with the
+                                     // actual user name if it will be available
+            message   : that._content.get('value')
+          });
+          CC_SERVICE.addComment(that.bugHash, commentData);
+          topic.publish('showComments', that.bugHash, that.sender);
+        }
+      });
+    },
+
+    postCreate : function () {
+      this.addChild(this._content);
+      this.addChild(this._commentBtn);
+    }
+  });
+
+  var Comment = declare(ContentPane, {
+    constructor : function (args) {
+      dojo.safeMixin(this, args);
+
+      var that = this;
+
+      //--- Header section ---//
+
+      this._header = new ContentPane();
+      var header = dom.create('div', { class : 'header'}, this._header.domNode);
+
+
+      var avatar = dom.create('div', { class : 'avatar'}, header);
+      style.set(avatar, 'background-color', strToColor(this.author));
+
+      var avatarLabel = this.author.charAt(0).toUpperCase();
+      var avatarContent = dom.create('div', {
+        class : 'avatar-content', innerHTML: avatarLabel }, avatar);
+
+
+      var vb = dom.create('div', { class : 'vb'}, header);
+
+      dom.create('span', { class : 'author', innerHTML: this.author }, vb);
+
+      var time = timeAgo(new Date(this.time));
+      dom.create('span', { class : 'time', innerHTML: time }, vb);
+
+      //--- Comment operations (edit, remove) ---//
+
+      var operations = dom.create('div', { class : 'operations'}, header);
+      dom.create('span', {
+        class   : 'customIcon edit',
+        onclick : function () { that.edit(); }
+      }, operations);
+
+      dom.create('span', {
+        class   : 'customIcon delete',
+        onclick : function () { that.remove(); }
+      }, operations);
+
+      //--- Message section ---//
+
+      this._message = new ContentPane({
+        class   : 'message',
+        content : this.message
+      });
+
+      //--- Remove comment confirmation dialog ---//
+
+      this._removeDialog = new ConfirmDialog({
+        title     : 'CodeChecker - Remove comment',
```

Comment:
I think putting `CodeChecker` here is rendundant. The message is really short which already makes the box small.